reference/snapshot.md

```diff
@@ -0,0 +1,276 @@
+---
+Title: OpenEBS Volume Snapshot
+Sidebar: Volume snapshot
+---
+
+
+**Volume snapshots** are copies of a persistent volumes at a specific point in time. They can be used to restore a volume to a previous state or create a new volume. Mayastor provides support for industry standard copy-on-write (COW) snapshots, which is a popular methodology for taking snapshots by keeping track of only those blocks that have changed.
+Mayastor incremental snapshot capability enhances data migration and portability in Kubernetes clusters across different cloud providers or data centers. Using standard kubectl commands, you can seamlessly perform operations on snapshots and clones in a fully Kubernetes-native manner.
+
+Use cases for volume snapshots include:
+
+- Efficient replication for backups.
+- Utilization of clones for troubleshooting.
+- Development against a read-only copy of data.
+
+
+Volume snapshots allow the creation of read-only incremental copies of volumes, enabling you to maintain a history of your data. These volume snapshots possess the following characteristics:
+- **Consistency**: The data stored within a snapshot remains consistent across all replicas of the volume, whether local or remote.
+- **Immutability**: Once a snapshot is successfully created, the data contained within it cannot be modified.
```

Comment:
@avishnu should we also mention volumes become thin provisioned once a snapshot is created? IMHO this is a very important nuance